Core1Platform/AdministratingCore1/03_ManageGroups.md

```diff
@@ -0,0 +1,186 @@
+- You cannot remove a user from the ActindoAdmingroup in a sandbox of which he or she is the owner. <!--- Kann man den owner ändern? -->
+- You cannot remove a user from all groups in an account in which the user has been created. In this case, you have to inactivate him&frasl;her, see [Edit user](./02_ManageUsers.md#edit-user).   
+
+Note that it is not possible to remove a group assignment by delimiting the validity end date to a specific date. If you want to change the validity period, you must remove the group assignment and create a new one with a limited validity end date.
+
+
+#### Prerequisites
+
+- You are granted to remove the user assignment from a group.
+
+#### Procedure
+
+*Settings > Users and groups > Tab GRUPPENVERWALTUNG (GROUP MANAGEMENT)*
+
+![Group management](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupManagment.png "[Group management]")
+
+1. Select the group, from which you want to remove a user.   
+    The editing toolbar is displayed.
+
+   ![Group selected](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupSelected.png "[Group selected]")
+
+2. Click the ![Edit](../../Assets/Icons/Edit01.png "[Edit]") (Edit) button.
+    The *Edit group "Group name"* view is opened. By default, the *Users* tab is displayed. 
+
+   ![Group selected](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupManagementUsers.png "[Group selected]")
+
+3. Select a user.   
+    The editing toolbar is displayed.
+
+   ![User selected](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupUserSelected.png "[User selected]")
+
+4. Click the ![Delete](../../Assets/Icons/Trash03.png "[Delete]") (Delete) button.   
+    The selected user is removed from the list.
+
+5. Click the [SAVE] button.   
+    The *GRUPPENVERWALTUNG (GROUP MANAGEMENT)* view is displayed again.
+
+
+## Remove user group
+
+If a user group is no longer needed, you can remove it. Note, groups are removed regardless of whether users are still assigned.
+
+#### Prerequisites
+
+- You are granted to delete groups.
+
+#### Procedure
+
+*Settings > Users and groups > Tab GRUPPENVERWALTUNG (GROUP MANAGEMENT)*
+
+![Group management](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupManagment.png "[Group management]")
+
+1. Check whether users are assigned the group. To do this, select the checkbox of the user group you want to remove.   
+    The editing toolbar is displayed.
+
+    ![Group selected](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupSelected.png "[Group selected]")
+
+2. Click the ![Edit](../../Assets/Icons/Edit01.png "[Edit]") (Edit) button.
+    The *Edit group "Group name"* window is opened.  
+
+    ![Edit group](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupEditNoUsers.png "[Edit group]")
+
+3. Check whether users are assigned the group. If necessary, remove the assignment, see [Remove a user from a group](#remove-a-user-from-a-user-group).
```

Comment:
Link funktioniert nicht, da Header "Remove user from a group" heißt.
Wir schreiben in der Regel die Header mit Artikeln (also, Delete a user from a group, usw.), wenn es um Prozeduren geht (also, Integration und Operation). Wenn es sich auf ein Fenster/eine Ansichtname bezieht, also User interface, nehmen wir den Fenster-/Ansichtnamen wie es ist, z.B. Create view (and nicht Create A view). So ist die Doku konsistenter und minimieren die potenzielle Fehler  bei der Verlinkung.

Reply:
sind a mal wieder seh eingängige Regeln, und so einheitlich..